users/api/http/requests.go

```diff
@@ -7,7 +7,9 @@
 
 package http
 
-import "github.com/mainflux/mainflux/users"
+import (
```

Comment:
Is this done by Go fmt? because you have only one line, I am not sure that these parenthesis are needed. but if gofmt is putting them, then OK.